commands/app.go

```diff
@@ -0,0 +1,79 @@
+package commands
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"unicode"
+
+	"github.com/pkg/errors"
+	"github.com/smallstep/certificates/authority"
+	"github.com/smallstep/certificates/ca"
+	"github.com/smallstep/cli/errs"
+	"github.com/urfave/cli"
+)
+
+// AppCommand is the action used as the top action.
+var AppCommand = cli.Command{
+	Name:   "start",
+	Action: appAction,
+	Flags: []cli.Flag{
```

Comment:
usage text? description?

Reply:
This is a virtual command for the cli.App, I basically moved the code to a new file. The description and all that is in the main.go